test/merchant_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/merchant'
+
+class MerchantTest < Minitest::Test
+  def test_it_exists
```

Comment:
You can organize a `def setup` at the top of a test file so you can move 
```
@m = Merchant.new({
                       :id => 5,
                       :name => "Turing School"
                       })
```

Makes things a little easier so you don't have to keep pasting a bunch